release/action.yaml

```
@@ -5,20 +5,23 @@ inputs:
     required: false
     description: "Perform checkout as first step of action"
     default: true
+  github-token:
+    required: true
+    description: "GitHub Personal Access Token that can checkout the consumer git repository as well as create tags/releases against it. e.g. 'secrets.GITHUB_TOKEN'"
   dry-run:
     required: false
     description: "Whether to run semantic release in `dry-run` mode. It will override the dryRun attribute in your configuration file"
   extra-plugins:
     required: false
     description: "Extra plugins for pre-install. You can also specify specifying version range for the extra plugins if you prefer."
 outputs:
-  new_release_published:
+  new-release-published:
     description: "Whether a new release was published"
-    value: ${{ steps.release.outputs.new_release_published }}
-  new_release_version:
+    value: ${{ steps.release.outputs.new-release-published }}
+  new-release-version:
     description: "Version of the new release"
-    value: ${{ steps.release.outputs.new_release_version }}
-  new_release_major_version:
+    value: ${{ steps.release.outputs.new-release-version }}
```

```suggestion
    value: ${{ steps.release.outputs.new_release_version }}
```
cycjimmy/semantic-release-action usins snake case

---

This has been addressed.